source/installation/install_postgres_server.rst

```diff
@@ -11,16 +11,64 @@
 Install PostgresSql server
 ##########################
 
-.. todo::
+On a linux environment, installing Postgresql server is very easy.
+Here follows the instructions for a debian based distribution (as Ubuntu).
 
-   the section "Install PostresSql database" must be written. Help appreciated :-)
-   
+.. code-block:: bash
+	
+	sudo apt-get update
+	sudo apt-get install postgresql
+	
+Having a look at the install messages, you will guess quickly which is the version installed.
+Anyway you can check this with the following code:
+
+.. code-block:: bash
+	
+	psql --version
+	>>> psql (PostgreSQL) 9.4.5
+
+To be able to add the unaccent extension to your database, 
+you will have to install the following package where x.x are the first 2 parts of the version, 
+9.4 in this example.
+
+.. code-block:: bash
+
+	sudo apt-get install postgresql-contrib-x.x
+	
+You are ready to play with Postgressql.
+	
 .. note:: 
 
    To avoid installation and configuration of a postgresql server, you may use `our docker image <https://registry.hub.docker.com/u/chill/database/>`_ to start and configure a database as decribed in the basic installation chapter.
    This solution can be used also in a production environment.
+      
    
-.. note::
+Install PostresSql database
+###########################
+
+Here follows as an example the instructions that has been used on Ubuntu 14.04 distribution to install the Chill database.
+Feel free to customize it following your preferences, but do not forget to enable the `unaccent` extension on your database.
+
+.. code-block:: bash
 
-   Installing unaccent extension on ther server is possible with the package `postgresql-contrib-9.x` (replace 9.x with your server version). The extension may be enabled with running `CREATE EXTENSION unaccent;` in the database, with a superuser account.
+	sudo su
+	su postgres
+	psql
+		CREATE USER chill_user WITH PASSWORD 'my_terrible_secret' CREATEDB;
```

Comment:
It is easier to do this : 

```
sudo su
su postgres
createuser --pwprompt username
createdb -O username dbname
#then install unaccent
psql -d dbname -c "CREATE EXTENSION unaccent;"
```

But the instructions you wrote are ok.

If you think it is easier to do everything in SQL, you do not have to exit psql and connect again to switch to the newly created db. Simply use `\c chill_db` to connect to the chill_db database.